search/search_service/api/swagger_doc/template.yml

```diff
@@ -167,14 +178,81 @@ components:
               type: integer
               description: 'dashboard last successful run time'
               example: 1568814420
+    FeatureFields:
+      type: object
+      properties:
+        id:
+          type: string
+          description: 'elasticsearch doc id'
+          example: '1234aaabbb'
+        feature_group:
+          type: string
+          description: 'group name which feature belongs to'
+          example: 'realtime_post_processed_group'
+        feature_name:
+          type: string
+          description: 'name of feature'
+          example: 'ml_feature_420_abcd'
+        version:
+          type: string
+          description: 'version as string'
+          example: 'v1.1.5'
+        key:
+          type: string
+          description: 'key format: {feature_group}/{feature_name}/{version}'
+          example: 'realtime_post_processed_group/ml_feature_420_abcd/v1.1.5'
+        total_usage:
+          type: int
+          description: 'total usage'
+          example: 0
+        status:
+          type: string
+          description: 'feature status'
+          example: 'active'
+        entity:
+          type: string
+          description: 'business entity described by this feature'
+          example: 'listing'
+        description:
+          type: string
+          description: 'feature description'
+          example: 'this feature has durations of active listings in seconds'
+        availability:
+          type: array
+          items:
+            type: string
+          description: 'list of databases where this feature is available'
+          example: [ 'hive', 'dynamo' ]
+        badges:
+          type: array
+          items:
+            type: object
+            properties:
+              tag_name:
+                type: string
+          description: 'list of feature badges'
+          example: [ { 'tag_name': 'badge1' }, { 'tag_name': 'badge2' } ]
+        tags:
+          type: array
+          items:
+            type: object
+            properties:
+              tag_name:
+                type: string
+          description: 'list of feature tags'
+          example: [ { 'tag_name': 'tag1' }, { 'tag_name': 'tag2' } ]
+        last_updated_timestamp:
+          type: integer
+          description: 'unixtime when the feature configuration was last updated'
+          example: 1568814420
     UserFields:
       type: object
       properties:
         id:
           type: string
           description: 'elasticsearch doc id'
           example: 'M81jD3cBdULZTSY96PSh'
-        name:
+        full_name:
```

Comment:
see https://github.com/amundsen-io/amundsen/commit/79792cdd8781c5e9e627f50ae5f4a5682f4d8003 for explanation